lib/new-package.js

```diff
@@ -455,11 +457,14 @@ function packageJson(options, packageName) {
       'update-examples': 'node maintenance/update-examples-from-preview.js',
       postinstall: 'elm-tooling install'
     },
-    dependencies: {
+    engines: {
+      node: '>=14.21.3'
+    },
+    devDependencies: {
```

Comment:
If they're only needed at development-time, I don't see why we should need them as dependencies, but if you'd rather revert them, that's fine as well.